src/System/BpmControl.cpp

```diff
@@ -0,0 +1,33 @@
+#include "HardwareInput.h"
+#include "BpmControl.h"
+
+#define ANALOG_MAX 1023
+#define ANALOG_FREQ 20000
+
+BpmControl::BpmControl (pin_size_t pinNum){
+    _pinNum = pinNum;
+}
+
+void BpmControl::init (){
+    pinMode(_pinNum, OUTPUT);
+    analogWriteRange(ANALOG_MAX);
+    analogWriteFreq(ANALOG_FREQ);
+};
+
+void BpmControl::setControlSignal (int32_t controlSignal){ 
```

Comment:
The spacing before the parens here and in the header don't follow the style of the rest of the supermileage codebase. The style guide is [here](https://www.notion.so/C-Style-Guide-f40ef59a0c3140148e1e578bfc61f1b8).

It might be good to set up a linter eventually.